app/components/Forms/AltDateTimeInput.tsx

```diff
@@ -3,8 +3,9 @@
 import React from "react";
 import PropTypes from "prop-types";
 import AltDateInput from "./AltDateInput";
+import { WidgetProps } from "react-jsonschema-form";
 
-function AltDateTimeInput(props) {
+function AltDateTimeInput(props: WidgetProps) {
```

Comment:
We've been using `const AltDateTimeInput: React.FunctionComponent<WidgetProps> = (props) =>` to ensure that the return type is also correct